### ISSUE TITLE
Overzealous test in `is_internet` script disconnects Kano from the internet

### ISSUE THREAD

**Issue by aspiwack** [KanoComputing/kano-toolset#212]
The [following lines in `is_internet`](https://github.com/KanoComputing/kano-toolset/blob/ae351b22ffda9c615b2493e0e2eb2019c64d2180/bin/is_internet#L15-L24) are faulty

``` sh
# Kanux provides networking through either Ethernet or Wireless
# Before trying to reach the internet we make sure that one of this devices
# is hard-wired (ethernet) or associated (wireless).
sudo /usr/sbin/ifplugstatus eth0 > /dev/null 2>&1
lnketh0=$?
sudo /usr/sbin/ifplugstatus wlan0 > /dev/null 2>&1
lnkwlan0=$?
if [ "$lnketh0" != "2" ] && [ "$lnkwlan0" != "2" ]; then
    exit 3
fi
```

I discovered them after I got a replacement dongle which registered as `wlan1`, causing `is_internet` to fail systematically (there is neither an `eth0` or a `wlan0`).

This causes the whole Kano deskop to malfunction as
1. A number of applications (_e.g._ youtube) check `is_internet` before getting started, hence refuse to start.
2. The network manager uses `is_internet` to establish its state, and will restart the wireless acquisition periodically, killing the current connection (so internet is actually turned off half of the time).

I don't know how to make a more accurate test that "at least one network interface is plugged in" (maybe an argumentless `ifplugstatus` piped with an appropriate `grep`). But if the exit code `3` is not specifically used by other programs, maybe just removing these lines would be an appropriate fix (as `fping` will fail if no interface is plugged in anyway).

**Comment by ieguiguren** [comment]
Same behavior here. 

I've switched the sd card to different Rasperries and during boot each adds a new device line to /etc/udev/rules.d/70-persistent-net.rules
I consider rather unnecessary to check the interface to know if there is internet connectivity or not. Just trying to reach a website via wget or curl is enough to verify if everything is setup correctly no matter if connection works via air or cable.

Attached you may find the version I'm actually using which solved the issue. Taking into account that this is for kids, and such script is for internal use, I don't think it's useful to check if the connectivity issue is in the link or ip layer.

[is_internet.txt](https://github.com/KanoComputing/kano-toolset/files/2656684/is_internet.txt)

Thanks for your time and effort.

**Comment by radujipa** [comment]
Thanks @ieguiguren, we managed to reproduce this and we'll be addressing this issue with the next release v4.2.0.

**Comment by radujipa** [comment]
@ieguiguren The tagged fixes have been made and released to both 3.16.1 Jessie and 4.2.0 Stretch.

Feel free to reopen if you're still having issues.
